privaterelay/models.py

```diff
@@ -14,6 +15,27 @@ def __str__(self):
         return 'Invitation for %s' % self.email
 
 
+def get_invitation(email=None, fxa_uid=None, active=False):
+    local_args = locals()
+    invitations = Invitations.objects.filter(
+        Q(email=email) | Q(fxa_uid=fxa_uid), active=active
+    ).order_by('date_added')
+    if invitations.count() < 1:
+        raise Invitations.DoesNotExist(
+            'get_invitation found no invitation for args: %s' % local_args
+        )
+    invitations = list(invitations)
+    oldest_invitation = invitations.pop(0)
+    for invite in invitations:
+        if oldest_invitation.email != invite.email:
+            oldest_invitation.email = invite.email
+        if oldest_invitation.fxa_uid != invite.fxa_uid:
+            oldest_invitation.fxa_uid = invite.fxa_uid
```

Comment:
```suggestion
        if invite.email and oldest_invitation.email != invite.email:
            oldest_invitation.email = invite.email
        if  invite.fxa_uid and oldest_invitation.fxa_uid != invite.fxa_uid:
            oldest_invitation.fxa_uid = invite.fxa_uid
```
Check to make sure that the `fxa_uid` and `email` have values and that we are updating the fields only when they have values.